python/py-praw/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-praw
+version             7.5.0
+supported_archs     noarch
+license             BSD
+
+python.versions     37 38 39
```

Comment:
`py-prawcore` had subports for these 3 Python versions (`37 38 39`), but 3.10 as well.  Any reason why that version is being skipped here?

Reply:
Hi @herbygillot,

When I looked at the dependencies for py-praw, it included py-update_checker as a dependency.  However, it looks like py-update_checker does not include support for Python 3.10, so I removed Python 3.10 from the list supported Python version for py-praw.  When py-update_checker is updated to include support for Python 3.10, I think we could include Python 3.10 support for py-praw. 

Thanks,

-ranga

Reply:
You can add a `310` subport to `py-update_checker` and see if it builds (which depends on whether there are `310` subports for all of `py-update_checker`'s dependencies).  If it does build, then you can add that as a separate commit with a description like `py-update_checker: add subport for Python 3.10`.  Then you would base this commit for `py-praw` on top of that.

Reply:
Hi @herbygillot,

I've submitted a separate pull request for adding Python 3.10 support to py-update_checker.  Once that is merged, I will update the Portfile for this port with support for Python 3.10.

Thanks,

-ranga

Reply:
OK, that's been merged.

Reply:
Hi @herbygillot,

I have updated the Portfile to include Python 3.10. 

Thanks,

-ranga

Reply:
OK great. Final minor nitpick is to update the commit message to say `py-praw` instead or `praw` and I think we're good here.